packages/react-components/src/components/InputBoxComponent.tsx

```diff
@@ -85,13 +85,10 @@ export const InputBoxComponent = (props: InputBoxComponentProps): JSX.Element =>
   );
 
   const mergedTextContainerStyle = mergeStyles(textContainerStyle, styles?.textFieldContainer);
-  const mergedTextFieldStyle = concatStyleSets(
-    textFieldStyle(isDarkThemed(theme) ? '#f1707b' : '#a80000', !!errorMessage, !!disabled),
-    {
-      fieldGroup: styles?.textField,
-      errorMessage: styles?.systemMessage
-    }
-  );
+  const mergedTextFieldStyle = concatStyleSets(textFieldStyle(isDarkThemed(theme) ? '#f1707b' : '#a80000'), {
```

Comment:
why are we removing `!!errorMessage, !!disabled`

Reply:
These are Boolean variables used to change border color based on state(error, disabled). Since we have removed border from textfield, there is no use of these variables.

Reply:
Should these conditions then now apply to the sendbox border we are creating?